ministryofjustice/laa-claim-non-standard-magistrate-fee-backend
# C rm457 538 setup auth

## Description of change
Add configuration elements for SAML authentication with Portal Dev in our Dev environment

## Link to relevant ticket
[Provider Build: Add Config for Dev SAML Authentication ](https://dsdmoj.atlassian.net/jira/software/c/projects/CRM457/boards/1192?selectedIssue=CRM457-538)

## Notes for reviewer

## How to manually test the feature
1. Run the pipeline to update environment variables and config 
2. Go to the test portal login [here](https://portal.dev.legalservices.gov.uk/oamfed/idp/initiatesso?providerid=new-crm7) and login to portal (see [ticket](https://dsdmoj.atlassian.net/browse/TA-2313) for instructions on where to get login details) 
3. Once logged in, check that login redirects to the dev app's claim page with a specific user logged in via devise (look at the user indicated in the nav bar)
